test/testdoT.js

```diff
@@ -61,4 +61,15 @@ describe('doT', function(){
 		});
 	});
 
+	describe('whitespace stripping', function() {
+		function assert_match(actual, expected) {
+			assert(actual.match(expected), JSON.stringify(actual) + " did not match " + expected);
+		}
+		it('should replace stretches of whitespace with one whitespace', function() {
+			assert_match(doT.template('<div>    </div>')(), /^<div>\s<\/div>$/);
```

Comment:
I understand the way the functionality works, but tests should *always* be as simple and explicit as possible. Both tests should be updated. In my opinion, the regex gives the illusion that there is some uncertainty/room for variation when that is not the case.